packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -390,7 +390,7 @@ export function transformComplexTypes(
             ? 'CollectionField'
             : isComplex
             ? formattedTypes[parsedType] + 'Field'
-            : edmToComplexPropertyType(type),
+            : edmToComplexPropertyType(type, true),
```

Comment:
This seems wrong. Why it is always nullable?

Reply:
Before, everything was non-nullable, because the nullable fields didn't exist. This is the legacy code, so I decided not to touch it. Do you think this is ok?